.tekton/tag-semver.yaml

```diff
@@ -0,0 +1,66 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: cachi2-on-tag-created
+  annotations:
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-event: '[create]'
```

Comment:
This pipeline looks like a draft, but I have a possibly cool suggestion:

Maybe we could use `on-cel-expression` and check if the ref_type is a tag right away? https://pipelinesascode.com/docs/guide/authoringprs/#advanced-event-matching

Not sure if that would work, since the docs don't mention that `ref_type` would be an available field - but then again, the docs don't even mention that `create` would be an available event :confused:

Reply:
FWIW it looks like PaC releases itself like this https://github.com/openshift-pipelines/pipelines-as-code/blob/main/.tekton/release-pipeline.yaml

Reply:
This was indeed a draft, and it was commited by accident :face_exhaling: 

I saw a mention of Gihub events somewhere in the Tekton docs (not PaC), and took a wild shot to see if this would be triggered, but it really is not. So, no luck here.

Thanks for linking the PaC release approach, might be a inspiration for us to set up ours.